### ISSUE TITLE
Save Data Erased 

### ISSUE THREAD

**Issue by Mikeylovesretro** [tpunix/SAROO#128]
So I'm sure I used in-game saves for several games like Albert Odessy and Dracula X though it did not show up when I went back. I had to create a new in-game save. Is it just me or has this happened to anyone else. 

Thanks!

**Comment by Evil-Vegan** [comment]
Hello.

I tried both games with in-game saves and could not replicate your issue. _Albert Odyssey: Legend of Eldean_ is the Albert Odyssey - Legend of Eldean (USA) (RE) BIN/CUE. One potential difference is my SD card is formatted to exFAT. It seems a lot of problems happen to cards formatted to FAT32.

US NTSC VA1 Saturn
SanDisk 256GB Extreme MicroSDXC UHS-I (exFAT)
Redump BIN/CUE

MCU:240515
SS:240515
FPGA:05

**Comment by Mikeylovesretro** [comment]
Yeah it might of been a fluke I'm on exfat which is better for sure.

**Comment by blorp-style** [comment]
Hi, yes I did have all of my save data suddenly erased. I'm not totally sure why though. I lost about five hours of progress into Shining the Holy Ark, and about 5 hours of progress into Grandia (English patched version). It's possible I turned off the system while it was still saving, though I was unable to replicate the issue using this method

MCU:240627
SS:240627
FPGA:05
Sandisk 64 GB formatted to exFAT

Here is what my save file looked like once I noticed:
[SS_SAVE.zip](https://github.com/user-attachments/files/16325061/SS_SAVE.zip)

**Comment by Mikeylovesretro** [comment]
> Hi, yes I did have all of my save data suddenly erased. I'm not totally sure why though. I lost about five hours of progress into Shining the Holy Ark, and about 5 hours of progress into Grandia (English patched version). It's possible I turned off the system while it was still saving, though I was unable to replicate the issue using this method
> 
> MCU:240627 SS:240627 FPGA:05 Sandisk 64 GB formatted to exFAT
> 
> Here is what my save file looked like once I noticed: [SS_SAVE.zip](https://github.com/user-attachments/files/16325061/SS_SAVE.zip)

This is after updating to ver firm 0.6 ?

**Comment by blorp-style** [comment]
> > Hi, yes I did have all of my save data suddenly erased. I'm not totally sure why though. I lost about five hours of progress into Shining the Holy Ark, and about 5 hours of progress into Grandia (English patched version). It's possible I turned off the system while it was still saving, though I was unable to replicate the issue using this method
> > MCU:240627 SS:240627 FPGA:05 Sandisk 64 GB formatted to exFAT
> > Here is what my save file looked like once I noticed: [SS_SAVE.zip](https://github.com/user-attachments/files/16325061/SS_SAVE.zip)
> 
> This is after updating to ver firm 0.6 ?

Correct. It was working for a while then everything got erased.
